semopenalex-app/data/templates/Template%3Ahttp%3A%2F%2Fwww.w3.org%2F2004%2F02%2Fskos%2Fcore%23Concept.html

```diff
@@ -0,0 +1,506 @@
+    background-repeat: no-repeat no-repeat;
+    background-position: center center;
+    background-size: cover;
+    width: 12px;
+    height: 12px;
+  }
+  .publications-since__label, .publications-count__label {
+    font-size: 1.3rem;
+  }
+  .publications-since__value, .publications-count__value {
+    font-size: 3.2rem;
+    font-weight: 700;
+  }
+  .publications-since__label, .publications-since__value {
+    color: #000000;
+  }
+  .publications-count__label, .publications-count__value {
+    color: var(--soa-color-green);
+  }
+</style>
+<div class="soa">
+  <div class="header">
+    <mp-label iri='{{page-resource}}' class="header__title-main"></mp-label>
+  </div>
+  <div class="row cards">
+    <div class="card publication-chart">
+      <div>
+        <div>
+          <div class="publications-since__label">Publications since</div>
+          <semantic-query query='
+            SELECT ?publicationYear WHERE {
+              {
+                SELECT * WHERE {
+                  ?publication <https://semopenalex.org/property/hasConcept> <{{page-resource}}> .  
+                }
+                LIMIT 1001
+              }
+              ?publication <http://purl.org/spar/fabio/hasPublicationYear> ?publicationYear .  
+            } 
+            ORDER BY ASC(?publicationYear)
+            LIMIT 1' template='{{>tmp}}'>
+            <template id="tmp">
+              {{#each bindings}}
+                <div class="publications-since__value">{{publicationYear.value}}</div>
+              {{/each}}
+            </template>
+          </semantic-query>
+        </div>
+        <div>
+          <div class="publications-count__label">Publications</div>
+          <semantic-query query='
+            SELECT (COUNT (?publication) as ?count) WHERE {
+              ?publication <https://semopenalex.org/property/hasConcept> <https://semopenalex.org/concept/C71924100> .  
+            }
+          ' template="{{> tmp}}">
+            <template id="tmp">
+              {{#each bindings}}
+                <div class="publications-count__value">{{count.value}}</div>
+              {{/each}}
+            </template>
+          </semantic-query>
+        </div>
+      </div>
+      <semantic-chart type="bar" query="
+        SELECT ?year (COUNT(?publication) as ?count) WHERE {
+          {
+            SELECT * WHERE {
+              ?publication <https://semopenalex.org/property/hasConcept> <{{page-resource}}>
+            }
+            LIMIT 1001
+          }
+          ?publication <http://purl.org/spar/fabio/hasPublicationYear> ?year .
+        }
+        GROUP BY ?year
+        ORDER BY ASC(?year)
```

Comment:
let's use `countsByYear` predicate. x-axis to show year and y-axis to show publications.